coreneuron/CMakeLists.txt

```diff
@@ -18,28 +18,35 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 # =============================================================================
 file(GLOB_RECURSE CORENEURON_HEADER_FILES "*.h*")
 file(GLOB_RECURSE CORENEURON_TEMPLATE_FILES "*.ipp")
-file(GLOB_RECURSE CORENEURON_CODE_FILES
+file(GLOB CORENEURON_CODE_FILES
+                  "apps/main1.cpp"
+                  "gpu/*.c*"
+                  "io/*.c*"
+                  "mechanism/*.c*"
+                  "mpi/*.c*"
+                  "network/*.c*"
                   "nrniv/*.c*"
-                  "nrnmpi/*.c*"
                   "nrnoc/*.c*"
-                  "nrnomp/*.c*"
-                  "utils/*.c*")
+                  "permute/*.c*"
+                  "sim/*.c*")
```

Comment:
minor : all of the files are now .cpp (except .cu), so above could be .cpp instead of .c*?

Reply:
Yeah, I saw this, but wasn't sure if i should do the switch...